ams/api/src/main/java/com/netease/arctic/ams/api/Action.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.ams.api;
+
+import org.apache.iceberg.relocated.com.google.common.collect.Sets;
+
+import java.util.Collections;
+import java.util.Set;
+
+public enum Action {
+  MINOR_OPTIMIZING("minor-optimizing", 0),
+  MAJOR_OPTIMIZING("minor-optimizing", 1),
+  EXTERNAL_OPTIMIZING("external-optimizing", 2),
+  REFRESH_SNAPSHOT("refreshing", 10),
```

Comment:
Why not call it `refreshing-snapshot` ... ?

Reply:
I modify this to REFRESH_METADATA cause refreshing should be a more complex operation including all kinds of metadata